spec/rbnacl/signatures/ed25519/signing_key_spec.rb

```diff
@@ -31,6 +31,10 @@
     expect(subject.keypair_bytes).to eq signing_keypair
   end
 
+  it "can be converted to curve25519 private key" do
+    expect(subject.to_private_key).to be_a_kind_of(RbNaCl::PrivateKey)
+  end
```

Comment:
It'd be good to find some test vectors and ensure the output is correct for both methods

Reply:
added test vectors (generated by `pynacl`).